priv/templates/phx.gen.live/core_components.ex

```diff
@@ -155,38 +130,23 @@ defmodule <%= @web_namespace %>.CoreComponents do
       <.button>Send!</.button>
       <.button phx-click="go" class="ml-2">Send!</.button>
   """
-  attr :type, :string, default: nil
   attr :class, :string, default: nil
-  attr :rest, :global, include: ~w(disabled form name value)
-
+  attr :rest, :global
   slot :inner_block, required: true
 
   def button(assigns) do
     ~H"""
-    <button type={@type} class={[button_classes(), @class]} {@rest}>
+    <button class={["btn", @class]} {@rest}>
       {render_slot(@inner_block)}
     </button>
     """
   end
 
-  @doc """
-  Returns the default button classes.
-
-  ## Examples
-
-      <.link class={button_classes()} href={~p"/items/new"}>
-        New Item
-      </.link>
-  """
-  def button_classes do
-    "inline-block phx-submit-loading:opacity-75 rounded-lg bg-zinc-900 hover:bg-zinc-700 py-2 px-3 text-sm font-semibold leading-6 text-white active:text-white/80"
-  end
-
   @doc """
   Renders an input with label and error messages.
 
   A `Phoenix.HTML.FormField` may be passed as argument,
-  which is used to retrieve the input name, id, and values.
+  which is used to retrieve the input name, values, and errors.
```

Comment:
I changed this line because I removed the IDs but I believe you decided to keep them.

I would still vote to remove the built-in IDs and people add them for the components that are necessary? Your call though.

Reply:
I'm strongly voting for keeping ids. Those are really useful when writing browser tests.

Reply:
Ack!